### PR TITLE
Added a build-manifest task

### PR DESCRIPTION
**Pull request by syedriko**
Added a build-manifest task as the build-container runs as a multi-platform build.

### CODE REVIEW

File: .tekton/lightspeed-rag-content-pull-request.yaml [hunk 1]
```diff
@@ -208,7 +208,7 @@ spec:
       - name: PLATFORM
         value: linux-c4xlarge/amd64
       - name: IMAGE
-        value: $(params.output-image)
+        value: $(params.output-image)-amd64
```
[the remark this answers is not in the document]
Comment (MartinBasti) [reply .tekton/lightspeed-rag-content-pull-request.yaml:211]: ah nevermind, I was looking at wrong task :D

**Comment by syedriko** [comment]
/override "Red Hat Konflux / ols-e2e-tests-rosa / lightspeed-rag-content"
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-rag-content"

**Pull request merged**